wifi-subsys/components/network/test/test_https.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "httpsclient.h"
+#include "unity.h"
+#include "wifi.h"
+#include "storage.h"
+
+TEST_CASE("https request", "[https]") {
+    esp_err_t err;
+    nvs_init();
+    set_default_credentials();
+    wifi_init();
+    vTaskDelay(10000 / portTICK_PERIOD_MS);
```

Comment:
If you handle the flow of the above three functions this is not necessary, anyway we must run away of adding delays, sometimes it can hide future problems or bugs